crypto/fipsmodule/sha/sha3_test.cc

```diff
@@ -26,6 +26,18 @@ class SHA3TestVector {
     uint8_t digest[SHA3_256_DIGEST_LENGTH];
     EVP_MD_CTX* ctx = EVP_MD_CTX_new();
 
+    // Test SHA3 when |experimental_unstable_enable_sha3| is disabled
+    // |experimental_unstable_enable_sha3| is desabled by default
+    // SHA3 should fail when |experimental_unstable_enable_sha3| is disabled
+    if (*experimental_unstable_enable_sha3_get() == 0){
```

Comment:
Can we assume this is true when starting this test?

Reply:
I am not sure where exactly and how is this variable initialized to 0. By testing, yes, it is always zeroed at the beginning but I could not find the initialization in the code (something to do with this magical macro). I am not sure, if a users enables sha3 and then runs the tests, wouldn't be in this case not 0?